api/routes.js

```diff
@@ -189,19 +189,51 @@ module.exports = function(app, debug){
 
     // complex queries
 
-    app.get('/currentPlaceMeasurements/:type', function(req, res){
+    // get latest measurement of one type for one place
+    app.get('/placeLatestMeasurement/:place/:type', function(req, res){
         var type = req.params.type;
+        var place = req.params.place;
 
-        database.complexQueries.currentPlaceMeasurements(type)
+        database.complexQueries.placeLatestMeasurement(place, type)
         .then(function(data){
             res.status(200).send(data);
         })
         .catch(function(error){
-            console.log('error in /currentPlaceMeasurements', error);
-            res.status(500).send('Couldn\'t get current live affluence database');
+            console.log('error in /placeLatestMeasurement', error);
+            res.status(500).send('Error in /placeLatestMeasurement/:place/:type');
         });
     });
 
+    // get latest measurement of one type for all places
+    app.get('/placesLatestMeasurement/:type', function(req, res){
```

Comment:
Since `/placesLatestMeasurement` and `sensorsLatestMeasurement` don't do exactly the same kind of thing (the former gets measurements for ALL places, whereas the latter gets measurement for SOME sensors).
Maybe call the first one `allPlacesLatestMeasurement` ?